items/weapons.xml

```diff
@@ -1591,9 +1591,9 @@
   </CraftedItem>
   <CraftedItem id="crpg_morningstar_mace_t3" name="{=6EWm09LY}Morningstar" crafting_template="crpg_Mace" modifier_group="mace">
     <Pieces>
-      <Piece id="crpg_morningstar_mace_t3_blade" Type="Blade" />
-      <Piece id="crpg_morningstar_mace_t3_handle" Type="Handle" />
-      <Piece id="crpg_morningstar_mace_t3_pommel" Type="Pommel" />
+      <Piece id="crpg_morningstar_mace_t3_blade" Type="Blade" scale_factor="105" />

```

Comment:
it's going to look funny . I'd increase the blade scale factor up to 120. Reduce the weight accordingly so it does hurt swing speed.

Reply:
Reworked scale factor in most recent commit.